read-fonts/src/tables/cmap.rs

```diff
@@ -46,6 +59,81 @@ impl Cmap<'_> {
         None
     }
 
+    /// Returns the index, encoding record and subtable for the most
+    /// comprehensive mapping available.
+    ///
+    /// Comprehensive means that tables capable of mapping the Unicode full
+    /// repertoire are chosen over those that only support the basic
+    /// multilingual plane. The exception is that symbol mappings are
+    /// preferred above all others
+    /// (see <https://github.com/harfbuzz/harfbuzz/issues/1918>).
+    pub fn best_subtable(&self) -> Option<(u16, EncodingRecord, CmapSubtable<'a>)> {
```

Comment:
`find(platform, encoding)` - which I think is find first on the off chance there are many - seems unopinionated and general purpose, entirely reasonable to add to read-fonts.

best seems like it might be more subjective. Is this definitely the only reasonable definition of best or is it an opinionated best?

Reply:
I believe there's no good reason to prefer some other approach.